File: Basic/Hello World/C++/test/HelloTest.cpp
#include "../src/Hello.cpp"
#include <gtest/gtest.h>

using namespace std;

TEST(HelloWordTest, isHelloWord) {

  Hello hello;

  ASSERT_EQ("Hello World", hello.sayHello() );
}

int main(int argc, char **argv){
  testing::InitGoogleTest(&argc, argv);
  return RUN_ALL_TESTS();
}

File: Basic/Hello World/JavaScript/test/hello-test.js
var chai = require('chai');
var expect = chai.expect; // we are using the "expect" style of Chai
var Hello = require('../src/hello');


describe('HelloTest', function() {
  it('sayHello() should return Hello World', function() {
    var hello = new Hello([]);
    expect(hello.sayHello()).to.equal('Hello World');
  });
});
